docs/pages/_app.js

```diff
@@ -212,6 +224,18 @@ function Analytics() {
   return null;
 }
 
+let reloadInterval;
+
+// Avoid infinite loop when "Upload on reload" is set in the Chrome sw dev tools.
+function lazyReload() {
+  clearInterval(reloadInterval);
+  reloadInterval = setInterval(() => {
+    if (document.hasFocus()) {
+      window.location.reload();
+    }
+  }, 100);
```

Comment:
0 will probably also work since you just want the current stack to be empy.

Reply:
I just copied this file from the core and pasted it here. To be honest, I don't know what this function does.